teamwork/templates/base.html

```diff
@@ -1,23 +1,78 @@
-{% load static %}
+  {% load static %}
 
-<!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
+  <!DOCTYPE html>
+  <html>
+    <head>
+      <meta charset="utf-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1">
+      <title>
+      {% block title %}
+          MyProjects
+      {% endblock %}
+      </title>
+      <meta name="description" content="Website for managing multiple projects in one place.">
+      <!-- Bootstrap CSS -->
+      <link rel="stylesheet" href="{% static 'bootstrap/css/bootstrap.min.css' %}">
 
-    <title>
-    {% block title %}
-        MyProjects
-    {% endblock %}
-    </title>
-    <meta name="description" content="Website for managing multiple projects in one place.">
-    <link rel="stylesheet" href="{% static 'bootstrap/css/bootstrap-theme.min.css' %}">
-    <link rel="stylesheet" href="{% static 'bootstrap/css/bootstrap.min.css' %}">
-    <link rel="stylesheet" href="{% static 'css/base.css' %}">
-  </head>
-  <body>
-    {% block content %}
-    {% endblock %}
-  </body>
-</html>
+      <!-- Custom CSS -->
+      <link rel="stylesheet" href="{% static 'css/main.css' %}">
+    </head>
+    <body>
+      <!-- Navigation bar -->
+      <nav class="navbar navbar-inverse navbar-static-top" role="navigation">
+                      <div class="container">
+                          <div class="navbar-header">
+                              <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
+                                  <span class="sr-only">Toggle navigation</span>
+                                  <span class="icon-bar"></span>
+                                  <span class="icon-bar"></span>
+                                  <span class="icon-bar"></span>
+                              </button>
+                              <p class="navbar-brand">{{ project.name }}</p>
+                          </div>
+                          <div class="navbar-collapse collapse">
+                              <ul class="nav navbar-nav">
+                                  <li class="active"><a href="{% url 'dashboard' user.username %}">Dashboard</a>
+                                  </li>
+                                  <li><a href="calendar">Calendar</a>
+                                  </li>
+                                  <li><button type="button" class="btn btn btn-default btn-success navbar-btn"> <span class="glyphicon glyphicon-plus"></span> Add new project </button>

```

Comment:
In class attribute the `btn` class is written twice. Also, `btn-default` and `btn-success` are meant to be exclusive so only one should be used.